wapitiCore/attack/mod_https_redirect.py

```diff
@@ -50,12 +63,20 @@ async def attack(self, request: Request, response: Optional[Response] = None):
         url = urlparse(request.url)
 
         is_http = False
+        http_request = Request(
+            request.url, request.method, request.get_params,
+            request.post_params, request.file_params, request.encoding,
+            request.enctype, request.referer, request.link_depth
+        )
         if url.scheme == "http":
```

Comment:
I think a `return` instructions is missing at the end of the `url.scheme == "http"` block.

Without it the same check is done twice